doc/engineering-reference/src/simulation-models-encyclopedic-reference-002/air-system-fans.tex

```diff
@@ -726,6 +726,108 @@ \subsubsection{Nomenclature for Simple Models}\label{nomenclature-for-simple-mod
 
 \({\dot Q_{toair}} = {H_{fan}} + \left( {{H_{motor}} - {H_{fan}}} \right) \cdot {f_{motortoair}}\) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ()
 
+\subsection{Fan Energy Index}\label{fan-energy-index}
+
+Fan Energy Index (FEI) is an energy-efficiency metric for fans including their motors and drives. This metric provides a standardized and consistent basis for comparing fan energy performance across fan types and sizes at a given fan operating point. Fan specifiers can use it to understand and communicate fan-efficiency design intent, while legislative and regulatory bodies can use it to define energy-efficiency requirements of fans.
```

Comment:
Add: "FEI is reported in the EquipmentSummary table report in the Fans subtable."